integration-tests/project-folder/sources/meta-fixtures/rename.bb

```diff
@@ -0,0 +1,13 @@
+foo='foo'
```

Comment:
for clarity, do you want to change the content of the variable to be 'bar'? Or are you testing that the variable content is not detected as a symbol?

Reply:
Yes, I am testing the variable content is not detected as a symbol (thought it is quite unlikely to happen). Only the variable name is renamed.